designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ScenarioActivityApiEndpoints.scala

```diff
@@ -0,0 +1,235 @@
+package pl.touk.nussknacker.ui.api
+
+import derevo.circe.{decoder, encoder}
+import derevo.derive
+import pl.touk.nussknacker.engine.api.process.{ProcessName, VersionId}
+import pl.touk.nussknacker.restmodel.{BaseEndpointDefinitions, BusinessError}
+import pl.touk.nussknacker.restmodel.BaseEndpointDefinitions.SecuredEndpoint
+import pl.touk.nussknacker.security.AuthCredentials
+import pl.touk.nussknacker.ui.process.repository.DbProcessActivityRepository.{
+  Attachment => DbAttachment,
+  Comment => DbComment,
+  ProcessActivity => DbProcessActivity
+}
+import pl.touk.nussknacker.ui.server.HeadersSupport.FileName
+import sttp.model.StatusCode.{NotFound, Ok}
+import sttp.model.{HeaderNames, MediaType}
+import sttp.tapir.EndpointIO.Example
+import sttp.tapir._
+import sttp.tapir.derevo.schema
+import sttp.tapir.json.circe.jsonBody
+
+import java.io.InputStream
+import java.time.Instant
+
+class ScenarioActivityApiEndpoints(auth: EndpointInput[AuthCredentials]) extends BaseEndpointDefinitions {
+
+  import ScenarioActivityApiEndpoints.Dtos._
+  import TapirCodecs.ContentDispositionCodec._
+  import TapirCodecs.HeaderCodec._
+  import TapirCodecs.ScenarioNameCodec._
+  import TapirCodecs.VersionIdCodec._
+  import TapirCodecs.ErrorsCodecs._
+
+  private val scenarioNotFoundErrorOutput: EndpointOutput.OneOf[BusinessError, BusinessError] = oneOf[BusinessError](
```

Comment:
you can declare is as `lazy val` and move it after endpoints definitions

Reply:
Done